test/jest-e2e.json

```diff
@@ -5,5 +5,9 @@
   "testRegex": ".e2e-spec.ts$",
   "transform": {
     "^.+\\.(t|j)s$": "ts-jest"
+  },
+  "moduleNameMapper": {
+    "@lib/testlib": "<rootDir>/../libs/testlib/src",
```

Comment:
Wanted to point out that here we use `<rootDir>/../...` because `<rootDir>` is set to be `<projectRoot>/test`, so to get back to `lib` we need to go back a directory.